File: SoloLearn Codes/Kaleidoscopes/Kaleidoscopes.java
import java.util.Scanner;

public class Kaleidoscopes
{
    public static void main(String[] args) {
        double kaleidoscope = new Scanner(System.in).nextDouble();
        double price = kaleidoscope * 107 / 20;
        System.out.println(kaleidoscope > 1 ? Math.round((price * 9 / 10) * 100.0) / 100.0 : Math.round(price * 100.0) / 100.0);
    }
}

File: SoloLearn Codes/Kaleidoscopes/Kaleidoscopes.cs
using System;


namespace Sololearn
{
    class Program
    {
        static void Main(string[] args)
        {
            double kalei = Convert.ToDouble(Console.ReadLine());
            double price = Math.Round(kalei * 107 / 20, 2, MidpointRounding.AwayFromZero);
            Console.WriteLine(kalei > 1 ? Math.Round(price * 9 / 10, 2, MidpointRounding.AwayFromZero) : price);
        }
    }
}

